app/javascript/products.vue

```diff
@@ -78,13 +92,15 @@ import Product from 'product.vue'
 import unconfirmedLinksOpenButton from 'unconfirmed_links_open_button.vue'
 import LoadingListPlaceholder from 'loading-list-placeholder.vue'
 import Pager from 'pager.vue'
+import elapsedDays from 'elapsed_days.vue'
 
 export default {
   components: {
     product: Product,
     'unconfirmed-links-open-button': unconfirmedLinksOpenButton,
     loadingListPlaceholder: LoadingListPlaceholder,
-    pager: Pager
+    pager: Pager,
+    elapsedDays
```

Comment:
```suggestion
    elapsedDays: ElapsedDays
```

コンポーネントの登録ってみたいな感じじゃなくて大丈夫なんでしたっけ？

Reply:
下記リンクを参考に`import elapsedDays from 'elapsed_days.vue'`の変数名を使ってコンポーネントを定義する省略記法を使用しました。

95行目をPascalCaseに変更しましたので、定義方法を`elapsedDays: ElapsedDays`に変更いたします！

https://jp.vuejs.org/v2/guide/components-registration.html#%E3%83%AD%E3%83%BC%E3%82%AB%E3%83%AB%E7%99%BB%E9%8C%B2